core/src/main/java/net/opentsdb/aura/metrics/core/TimeSeriesShardIF.java

```diff
@@ -59,4 +60,5 @@
 
   void flush();
 
+  MetricRegistry metricRegistry();
```

Comment:
We need not expose metric registry from the Shard. Instead we can pass that to the `NewDocstore` as a constructor argument.

Reply:
yeah but then we need to give it to the factory. I want to do that as part of a plugin refactoring I think with the TSDB code base.